### ISSUE TITLE
odd square character on damage display

### ISSUE THREAD

**Issue by splace** [vrml2/tnakgame#6]
![squares](https://user-images.githubusercontent.com/1234848/58567603-8b5ee480-822a-11e9-86d7-cbab3102ce7b.png)

**Comment by dwblaikie** [comment]
Yeah, I saw that too - that's usually what happens when a font doesn't support a certain character.

I don't know what interesting character I might've used (or might've ended up in the string by accident) there... maybe a smart quote or other odd Windows-ism.

**Comment by splace** [comment]
i think its the tank names being missing, the event passing stuff might be failing for array types?

**Comment by splace** [comment]
looking at the code i can't see the names ever being there, seems like it might be a bug in the browser, the square boxes being delimiting characters in the value of the mfstring js object, i will write some test code and report.

**Comment by splace** [comment]
<a class="issue-link js-issue-link" data-error-text="Failed to load issue title" data-id="449299746" data-permission-text="Issue title is private" data-hovercard-type="issue" data-hovercard-url="/create3000/titania/issues/109/hovercard" href="https://github.com/create3000/titania/issues/109" aria-label="#109, MF{type} toString() methods include carriage return">create3000/titania#109</a>
